### PR TITLE
Translate docs/concepts/configuration/overview.md in Japanese 

### LINKED ISSUE #15300: Translate docs/concepts/configuration/overview.md in Japanese
**This is a Feature Request**

**What would you like to be added**

Translate docs/concepts/configuration/overview.md in Japanese

**Why is this needed**

This document is not translated in Japanese.

**Comments**

Copy a file from en contents to content/ja/docs/concepts/configuration/overview.md and translate it into Japanese.

/language ja

### PR DESCRIPTION
**Pull request by inductor**
Close #15300

### CODE REVIEW

File: content/ja/docs/concepts/configuration/overview.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+- 意味がある場合は常に、関連オブジェクトを単一ファイルにグループ化します。多くの場合、1つのファイルの方が管理が簡単です。例として[guestbook-all-in-one.yaml](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/guestbook/all-in-one/guestbook-all-in-one.yaml)ファイルを参照してください。
+
+- 多くの`kubectl`コマンドがディレクトリに対しても呼び出せることも覚えておきましょう。たとえば、設定ファイルのディレクトリで `kubectl apply`を呼び出すことができます。
+
+- 不必要にデフォルト値を指定しないでください。シンプルかつ最小限の設定のほうがエラーが発生しにくくなります。
+
+- よりよいイントロスペクションのために、オブジェクトの説明をアノテーションに入れましょう。
+
+
+## "真っ裸"のPod に対する ReplicaSet、Deployments、およびJob
```
Comment (sftim) [inline content/ja/docs/concepts/configuration/overview.md]:
```suggestion
 ## "真っ裸"のPod に対する ReplicaSet、Deployment、およびJob
```

File: content/ja/docs/concepts/configuration/overview.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+---
+title: 設定のベストプラクティス
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+このドキュメントでは、ユーザーガイド、入門マニュアル、および例を通して紹介されている設定のベストプラクティスを中心に説明します。
+
+このドキュメントは生物です。このリストには載っていないが他の人に役立つかもしれない何かについて考えている場合、IssueまたはPRを遠慮なく作成してください。
```
Comment (nasa9084) [inline content/ja/docs/concepts/configuration/overview.md]:
```suggestion
このドキュメントは生ものです。このリストには載っていないが他の人に役立つかもしれない何かについて考えている場合、IssueまたはPRを遠慮なく作成してください。
```
[nit]

File: content/ja/docs/concepts/configuration/overview.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+{{% /capture %}}
+
+{{% capture body %}}
+## 一般的な設定のTips
+- 構成を定義する際には、最新の安定したAPIバージョンを指定してください。
+
+- 設定ファイルは、クラスタに反映される前にバージョン管理システムに保存されるべきです。これによって、必要に応じて設定変更を迅速にロールバックできます。また、クラスタの再作成や復元時にも役立ちます。
```
Comment (nasa9084) [inline content/ja/docs/concepts/configuration/overview.md]:
```suggestion
- 設定ファイルは、クラスターに反映される前にバージョン管理システムに保存されるべきです。これによって、必要に応じて設定変更を迅速にロールバックできます。また、クラスターの再作成や復元時にも役立ちます。
```

File: content/ja/docs/concepts/configuration/overview.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+## ラベルの使用
+
+- `{ app: myapp, tier: frontend, phase: test, deployment: v3 }`のように、アプリケーションまたはデプロイメントの__セマンティック属性__を識別する[ラベル](/docs/concepts/overview/working-with-objects/labels/)を定義して使いましょう。これらのラベルを使用して、他のリソースに適切なポッドを選択できます。例えば、すべての`tier：frontend`を持つPodを選択するServiceや、`app：myapp`に属するすべての`phase：test`コンポーネント、などです。このアプローチの例を知るには、[ゲストブック](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/guestbook/)アプリも合わせてご覧ください。
+
+セレクターからリリース固有のラベルを省略することで、Serviceを複数のDeploymentにまたがるように作成できます。 [Deployment](/docs/concepts/workloads/controllers/deployment/)により、ダウンタイムなしで実行中のサービスを簡単に更新できます。
+
+オブジェクトの望ましい状態はDeploymentによって記述され、その仕様への変更が_適用_されると、Deploymentコントローラは制御された速度で実際の状態を望ましい状態に変更します。
+
+- デバッグ用にラベルを操作できます。Kubernetesコントローラー（ReplicaSetなど）とServiceはセレクターラベルを使用してPodとマッチするため、Podから関連ラベルを削除すると、コントローラーによって考慮されたり、Serviceによってトラフィックを処理されたりすることがなくなります。既存のPodのラベルを削除すると、そのコントローラはその代わりに新しいPodを作成します。これは、「隔離」環境で以前の「ライブ」Podをデバッグするのに便利な方法です。対話的にラベルを削除または追加するには、[`kubectl label`](/docs/reference/generated/kubectl/kubectl-commands#label)を使います。
```
Comment (nasa9084) [inline content/ja/docs/concepts/configuration/overview.md]:
```suggestion
- デバッグ用にラベルを操作できます。Kubernetesコントローラー（ReplicaSetなど）とServiceはセレクターラベルを使用してPodとマッチするため、Podから関連ラベルを削除すると、コントローラーによって考慮されたり、Serviceによってトラフィックを処理されたりすることがなくなります。既存のPodのラベルを削除すると、そのコントローラーはその代わりに新しいPodを作成します。これは、「隔離」環境で以前の「ライブ」Podをデバッグするのに便利な方法です。対話的にラベルを削除または追加するには、[`kubectl label`](/docs/reference/generated/kubectl/kubectl-commands#label)を使います。
```

File: content/ja/docs/concepts/configuration/overview.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+{{< note >}}
+コンテナが常に同じバージョンのイメージを使用するようにするためには、そのコンテナイメージの[ダイジェスト](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier)を指定することができます(例:`sha256:45b23dee08af5e43a7fea6c4cf9c25ccf269ee113168c19722f87876677c5cb2`)。このダイジェストはイメージの特定のバージョンを一意に識別するため、ダイジェスト値を変更しない限り、Kubernetesによって更新されることはありません。
+{{< /note >}}
+
+{{< note >}}
+どのバージョンのイメージが実行されているのかを追跡するのが難しく、適切にロールバックするのが難しいため、本番環境でコンテナをデプロイするときは `：latest`タグを使用しないでください。
+{{< /note >}}
+
+{{< note >}}
+ベースイメージのプロバイダのキャッシュセマンティクスにより、`imagePullPolicy：Always`もより効率的になります。たとえば、Dockerでは、イメージが既に存在する場合すべてのイメージレイヤーがキャッシュされ、イメージのダウンロードが不要であるため、pullが高速になります。
```
Comment (nasa9084) [inline content/ja/docs/concepts/configuration/overview.md]:
```suggestion
ベースイメージのプロバイダーのキャッシュセマンティクスにより、`imagePullPolicy：Always`もより効率的になります。たとえば、Dockerでは、イメージが既に存在する場合すべてのイメージレイヤーがキャッシュされ、イメージのダウンロードが不要であるため、pullが高速になります。
```

**Comment by nasa9084** [comment]
LGTM
ping @MasayaAoyama

**Pull request merged**